tests/test_service.py

```diff
@@ -685,6 +685,25 @@ def test_rename_wishlist_product_invalid_content_type(self):
                                                             'product_name': 'surface_pro'},
                                                         headers={'content-type': 'text/plain'})
         self.assertEqual(resp.status_code, status.HTTP_415_UNSUPPORTED_MEDIA_TYPE)
+    
+    def test_remove_all_wishlist(self):
```

Comment:
Since we're not using the endpoint, can we remove the tests as well?

Reply:
Same as before, I want to try to make it work. Can we let it be for now?

Reply:
Oh ok. No worries. I thought this was the final one to be merged. My changes are already merged so I'm not blocked due to this anymore. Feel free to make all changes as you wish